.github/workflows/chain-docs.yml

```
@@ -2,19 +2,44 @@ name: chain-docs
 
 on:
   push:
-    branches: [ dev , master ]
+    branches: [ master ]
     paths:
     - 'docs/**'
     - 'ci/**'
     - '!README.md'
+  issue_comment:
+    types: [created, edited]
 jobs:
 
   build:
     name: Build
     runs-on: ubuntu-latest
+    if: github.event_name == 'push' || (github.event.comment.body == '/RUN' && (github.actor == 'allthatjazzleo' || github.actor == 'tomtau' || github.actor == 'lezzokafka' || github.actor == 'yihuang' || github.actor == 'calvinlauco' || github.actor == 'foreseaz' || github.actor == 'CeruleanAtMonaco' || github.actor == 'devashishdxt' || github.actor == 'leejw51crypto'))
```

`/RUN` ... How about `/staging` haha?

```suggestion
    if: github.event_name == 'push' || (github.event.comment.body == '/staging' && (github.actor == 'allthatjazzleo' || github.actor == 'tomtau' || github.actor == 'lezzokafka' || github.actor == 'yihuang' || github.actor == 'calvinlauco' || github.actor == 'foreseaz' || github.actor == 'CeruleanAtMonaco' || github.actor == 'devashishdxt' || github.actor == 'leejw51crypto'))
```